plugins/by-name/clipboard-image/default.nix

```diff
@@ -145,9 +145,12 @@ in
 
     extraConfigLua =
       let
-        setupOptions = {
-          default = processPluginOptions cfg.default;
-        } // cfg.filetypes // cfg.extraOptions;
+        setupOptions =
+          {
+            default = processPluginOptions cfg.default;
+          }
+          // cfg.filetypes
+          // cfg.extraOptions;
```

Comment:
Ah, so _any_ chained `//` updates will now cause indentation, even if it would be possible to fit the chain on a single line...

I wonder if this also applies when the attrset literal is the final value?

```nix
foo = a // b // {
  c = "c";
  d = "d";
};
```